### PR TITLE
Update example in readme

### PR DESCRIPTION
**Pull request by wsdjeg**

### CODE REVIEW

File: README_ZH_CN.md [hunk 1]
```diff
@@ -73,7 +73,7 @@
    " 正确指定路径用以设置runtimepath. 以下范例插件在sparkup/vim目录下
    Plugin 'rstacruz/sparkup', {'rtp': 'vim/'}
    " 避免插件名冲突,例如L9已存在,则可以指定
```
Comment (nfischer) [inline README_ZH_CN.md]: Does this comment need to be updated too?

**Comment by nfischer** [comment]
I might change it to something like, "Install L9 and avoid a Naming conflict if you've already installed a different version somewhere else." Maybe that will give the hint to new comers that they shouldn't use that line unless they want a second copy

File: README_ZH_CN.md (hunk 1 above)
Comment (wsdjeg) [inline README_ZH_CN.md]: sorry ,I forgot it,

**Comment by wsdjeg** [comment]
great,can you change it?or should I  push a commnit?

**Comment by nfischer** [comment]
I think you'll have to change it, since I can't do it unless I do a PR off your branch.

**Comment by wsdjeg** [comment]
Done :smile:

**Comment by wsdjeg** [comment]
there is something wrong with my branch.I need to re open it.

**Pull request closed without merging**